plugins/signals-react/README.md

```diff
@@ -0,0 +1,47 @@
+# @backstage/plugin-signals-react
+
+Welcome to the web library package for the signals plugin!
+
+Signals plugin allows backend plugins to publish messages to frontend plugins.
+
+## Getting started
+
+This plugin contains functionalities that help utilize the signals plugin. To get started,
+see installation instructions from `@backstage/plugin-signals-node`, `@backstage/plugin-signals-backend`, and
+`@backstage/plugin-signals`.
+
+There are two ways to utilize the signals plugin; either by using the hook or by directly using the API.
+
+## Using the hook
+
+By using the hook, unsubscribe is automatically taken care of. This helps to maintain only necessary amount
+of connections to the backend and also to allow multiple subscriptions using the same connection.
+
+Example of using the hook:
+
+```ts
+import { useSignalApi } from '@backstage/plugin-signals-react';
+
+const [message, setMessage] = useState<JsonObject | undefined>(undefined);
+useSignalApi('myplugin:topic', message => {
+  setMessage(message);
+});
```

Comment:
Tbh I think the hook form of this should probably just return the most recent signal or something like that directly. If one wants to use the callback it's a pretty small step to call `subscribe` directly.